tests/test_uploads.py

```diff
@@ -49,7 +49,8 @@ def test_chunked_stdin(httpbin_with_chunked_support):
     )
     assert HTTP_OK in r
     assert 'Transfer-Encoding: chunked' in r
-    assert r.count(FILE_CONTENT) == 2
+    # One original (request) + one encoded (httpbin's response, ascii)
+    assert r.count(FILE_CONTENT) + r.count(RAW_FILE_CONTENT) == 2
```

Comment:
The reason for these changes is that, httpbin mirrors the headers when it is responding. Since it is also mirroring the `Transfer-Encoding`, we treat this response as if it was chunked and use different delimiting techniques. So because of that, we stop formatting it as JSON, and not decode the actual unicode symbols (since that happens on formatting). Which I think is a plausible behavior, considering in this case, the server is the wrong side.